lib/mix/lib/mix/tasks/escript.install.ex

```diff
@@ -23,8 +23,9 @@ defmodule Mix.Tasks.Escript.Install do
       ~/.mix/escripts/foo
 
   For convenience, consider adding `~/.mix/escripts` directory to your
-  `PATH` environment variable. For more information, check the wikipedia
-  article on PATH: https://en.wikipedia.org/wiki/PATH_(variable)
+  `PATH` environment variable. For more information, check the [Wikipedia
+  article on PATH][PATH]
+  [PATH]: https://en.wikipedia.org/wiki/PATH_(variable)
```

Comment:
Because earmark can't handle it for now :) The documentation has stayed like this for the long time and I think it's fine; introducing this change is a workaround that we'll have to keep in mind, I think it's just simpler to leave it like this. Also, the purpose of Elixir's documentation is to be useful more than pretty, so this is fine IMO :) Wdyt?